Benchmark doc (#3394)
yugabyte/yugabyte-db: docs/content/latest/benchmark/ycsb/yugabytecql.md

You can do this by running the following commands.

```sh
cd $HOME
wget https://github.com/yugabyte/YCSB/releases/download/1.0/ycsb.tar.gz
tar -zxvf ycsb.tar.gz
cd YCSB
```

## Step 2. Start your database
Start the database using steps mentioned here: https://docs.yugabyte.com/latest/quick-start/explore-ysql/.

## Step 3. Configure your database
Create the Database and table using the cqlsh tool.
The cqlsh tool is distributed as part of the database package.

```sh
bin/cqlsh <ip> --execute "create keyspace ycsb"
bin/cqlsh <ip> --keyspace ycsb --execute 'create table usertable (y_id varchar primary key, field0 varchar, field1 varchar, field2 varchar, field3 varchar, field4 varchar, field5 varchar, field6 varchar,  field7 varchar, field8 varchar, field9 varchar);'
```

## Step 4. Configure YCSB connection properties
Set the following connection configurations in db.properties:

```sh
hosts=<ip>
port=9042
cassandra.username=yugabyte
```

The other configuration parameters like username, password, connection
parameters, etc. are described in detail at [this page](https://github.com/yugabyte/YCSB/tree/master/yugabyteCQL)

## Step 5. Running the workload

> Before you can actually run the workload, you need to "load" the data first.

Reviewer: ```suggestion
Before starting the workload, you will need to "load" the data first.
```

Author: Done.
